Compiled from "CWE546_Suspicious_Comment__HACK_09.java"
public class testcases.CWE546_Suspicious_Comment.CWE546_Suspicious_Comment__HACK_09 extends testcasesupport.AbstractTestCase {
  public testcases.CWE546_Suspicious_Comment.CWE546_Suspicious_Comment__HACK_09();
    Code:
       0: aload_0
       1: invokespecial #3                  // Method testcasesupport/AbstractTestCase."<init>":()V
       4: return

  public void bad() throws java.lang.Throwable;
    Code:
       0: ldc           #9                  // String This a test of the emergency broadcast system
       2: invokestatic  #11                 // Method testcasesupport/IO.writeLine:(Ljava/lang/String;)V
       5: return

  private void good1() throws java.lang.Throwable;
    Code:
       0: ldc           #9                  // String This a test of the emergency broadcast system
       2: invokestatic  #11                 // Method testcasesupport/IO.writeLine:(Ljava/lang/String;)V
       5: return

  private void good2() throws java.lang.Throwable;
    Code:
       0: ldc           #9                  // String This a test of the emergency broadcast system
       2: invokestatic  #11                 // Method testcasesupport/IO.writeLine:(Ljava/lang/String;)V
       5: return

  public void good() throws java.lang.Throwable;
    Code:
       0: aload_0
       1: invokevirtual #15                 // Method good1:()V
       4: aload_0
       5: invokevirtual #20                 // Method good2:()V
       8: return

  public static void main(java.lang.String[]) throws java.lang.ClassNotFoundException, java.lang.InstantiationException, java.lang.IllegalAccessException;
    Code:
       0: aload_0
       1: invokestatic  #23                 // Method mainFromParent:([Ljava/lang/String;)V
       4: return
}
